dokka-runners/dokka-gradle-plugin/src/main/kotlin/formats/DokkaFormatPlugin.kt

```diff
@@ -134,8 +134,9 @@ abstract class DokkaFormatPlugin(
                     // https://github.com/gradle/gradle/issues/27435)
                     dependenciesContainer.resolutionStrategy.eachDependency {
```

Comment:
FYI: Using `.eachDependency` to specify version could lead to Gradle complain. I've fixed similar issue inside `kotlin.git`. Proposed solution is to use constraints with required (or some other level) version.

Reply:
Constraints require a list of all Dokka dependencies, and I don't think we have this.

I think improving this is on Gradle to fix https://github.com/gradle/gradle/issues/15089